.github/workflows/create-lint-wf.yml

```diff
@@ -28,6 +28,8 @@ jobs:
           sudo ln -s /tmp/nextflow/nextflow /usr/local/bin/nextflow
 
       - name: Run nf-core/tools
+        env:
+          GITHUB_AUTH_TOKEN: ${{ secrets.nf_core_bot_auth_token }}
```

Comment:
As we're running `nf-core sync` _without_ the arguments to kick off a PR, it shouldn't be touching anything on GitHub. In fact it can't, as the test pipeline repo here won't have any remotes. So it's just a local sync only (and shouldn't do anything as it must be up to date by definition).

```suggestion
```